### PR TITLE
backward compatible support for login with password

### PR DESCRIPTION
**Pull request by dotansimha**
also, added support for skipping jsaccounts validation during execution

@davidyaha

### CODE REVIEW

File: packages/graphql-api/src/utils/authenticated-resolver.ts
Comment (davidyaha) [inline packages/graphql-api/src/utils/authenticated-resolver.ts:2]: Why is this good? I would expect users that does not want to use js-accounts verification to just not use the authenticated wrapper no?

**Comment by davidyaha** [comment]
@pradel You should take a look also before merging so you'll know if this affects your current work.

File: packages/graphql-api/src/utils/authenticated-resolver.ts
Comment (dotansimha) [reply packages/graphql-api/src/utils/authenticated-resolver.ts:2]: Because in some cases you want to skip the validation. 
For example: I am using the GraphQL execute as standalone in the server side, without the client executing anything, and I would like to execute it with a specific user.
Without the `skipJSAccountsVerification`, the JSAccounts will override it.
I want to be able to execute the specific schema with and without a user, but still to use `authenticated`

**Comment by dotansimha** [comment]
@pradel  please review :)

**Review by pradel — APPROVED** [review]
I think it should be good 👍

**Pull request merged**
